NGCHM/WebContent/javascript/DetailHeatMapDisplay.js

```diff
@@ -184,6 +184,7 @@ NgChm.DET.drawDetailHeatMap = function (mapItem, drawWin) {
 		NgChm.DMM.detailResize();
 		NgChm.DET.resizeOnNextDraw = false;
 	}
+	NgChm.DET.setViewPort(mapItem);
```

Comment:
@bmbroom I just discovered this line causes the following bug: if the detail pane is closed, so there is only a summary pane open, and a mouseout event occurs on the summary pane, errors related to a lack of detail pane are reported from PaneControl.js line 128, and DetailHeatMapDisplay.js line 865:

```
Uncaught TypeError: can't access property "clientWidth", detPane.pane is null
    setViewPort http://localhost/~marohrdanz/NG-CHM-second-copy/NGCHM/WebContent/javascript/DetailHeatMapDisplay.js?v=2.20.2:865
    drawDetailHeatMap http://localhost/~marohrdanz/NG-CHM-second-copy/NGCHM/WebContent/javascript/DetailHeatMapDisplay.js?v=2.20.2:187
    drawDetailTimeout http://localhost/~marohrdanz/NG-CHM-second-copy/NGCHM/WebContent/javascript/DetailHeatMapDisplay.js?v=2.20.2:89
...
```

Is this line critical to the bug fix of this pull request?